doc/related_projects.rst

```diff
@@ -237,9 +195,6 @@ Other packages useful for data analysis and machine learning.
 - `Pandas <https://pandas.pydata.org/>`_ Tools for working with heterogeneous and
   columnar data, relational queries, time series and basic statistics.
 
-- `theano <http://deeplearning.net/software/theano/>`_ A CPU/GPU array
```

Comment:
Theano is no longer developed unfortunately.

Reply:
Just as info: While MILA announced the end of Theano, pymc3 took over the maintenance (but without further features), see [this blog post](https://medium.com/@pymc_devs/theano-tensorflow-and-the-future-of-pymc-6c9987bb19d5). Last release was 1.04 in January 2019, see https://pypi.org/project/Theano.

Reply:
Indeed. I'm just saying that encouraging people to start using Theano now for DL is probably not very reasonable. But we can also leave it in as well, I don't have a strong opinion.

Reply:
Good point. I would remove the link.